src/pages/p2p/components/_hero.tsx

```diff
@@ -238,7 +238,7 @@ const Hero = () => {
         <BackgroundWrapper data={background}>
             <Wrapper>
                 <InformationWrapper height="unset" direction="column">
-                    <StyledHeader as="h1" weight={500}>
```

Comment:
just for clarification, We don't need `weight`?

Reply:
Header component already have a default weight as 500. So no need to add it again here